browser-sign-in/ios/browserSignIn-tvOS/Info.plist

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you delete tvOS target(with test target) from sample project?

Reply:
Deleted